crates/vm/levm/tests/tests.rs

```diff
@@ -3362,3 +3363,146 @@ fn create_on_create() {
     vm.execute();
     assert_eq!(vm.db.accounts.len(), 4);
 }
+
+#[test]
+fn callvalue_op() {
+    let address_that_has_the_code = Address::from_low_u64_be(0x42);
+    let value = U256::from(0x1234);
+
+    let operations = [Operation::Callvalue, Operation::Stop];
+
+    let tx_env = TxEnv {
+        transact_to: TransactTo::Call(address_that_has_the_code),
+        value,
+        ..Default::default()
+    };
+
+    let block_env = BlockEnv::default();
+
+    let mut db = Db::default();
+
+    db.add_account(
+        address_that_has_the_code,
+        Account::default()
+            .with_bytecode(ops_to_bytecde(&operations))
+            .with_balance(U256::MAX),
```

Comment:
```suggestion

```
I think this line isn't necessary because we don't care about the balance of the smart contract. Right? The same goes for some of the other tests.

Reply:
You're right, not sure why I added them.